game/hud/src/components/PlayerStatusComponent/index.tsx

```diff
@@ -169,39 +171,43 @@ class PlayerStatusComponent extends React.Component<PlayerStatusComponentProps,
                  valuePerPill={VALUE_PER_BODY_PARTY_PILL}
                  currentValue={this.props.playerStatus.health[BodyParts.RightArm].current}
                  maxValue={this.props.playerStatus.health[BodyParts.RightArm].maximum}
+                 flashThreshold={0}
```

Comment:
Body parts should have a flash threshold set as well as they have regen, you should be able to see that regen working by doing /ouch in game since abilities are disabled currently.